src/report.tex

```diff
@@ -1450,7 +1455,7 @@ \subsection{Conclusion}
 \\
 The thermal flux near the bioshield was examined and found to be significantly affected
 by the B$_4$C layer, this correlated with the shape of the thermal flux suggests that 
-it is the bioshield that is the source of thermalized neutrons in the \gls{epi} and \gls{upi}.
+it is the B$_4$C layer is effective at absorbing thermalized neutrons in the \gls{epi} and \gls{upi}.
```

Comment:
I didn't agree with Mike's comment here.  I think both things are true.  If the B4C layer has this effect, it can only be because the bioshield is a substantial source of thermal neutrons.